backend/src/game/turn.py

```diff
@@ -0,0 +1,33 @@
+import collections
+
+from backend.src.game import constants
+from backend.src.game.locations import Locations
+from backend.src.game.dice import DieValue
+
+
+def get_dice_count(dice):
+    return collections.counter(dice)
+
+
+def resolve_victory_points_dice(dice_counter):
+    victory_points = 0
+    for die, die_count in dice_counter.items():
+        if die in [DieValue.ONE, DieValue.TWO, DieValue.THREE]:
+            leftover_dice = die_count - 3
+            if leftover_dice >= 0:
+                victory_points += (die.value + leftover_dice)
```

Comment:
I don't like the if in an if in a for. But I also can't think of a better solution right now.

Reply:
Found a way to reduce the if statements, will upload shortly.